### PR TITLE
Remove HCAService - no longer used

### PR DESCRIPTION
**Pull request by tparesi**
#### Description:

Remove no longer used HCAService that was returning a no-op.

### CODE REVIEW

File: app/views/onboarding/LocationsPermissions.tsx [hunk 1]
```diff
@@ -30,13 +30,12 @@ import {
 const LocationsPermissions = (): JSX.Element => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
-  const { authSubscription, location } = useContext(PermissionsContext);
```
Comment (tparesi) [inline app/views/onboarding/LocationsPermissions.tsx:33]: Not sure if this is needed?

Comment (johnschoeman) [reply app/views/onboarding/LocationsPermissions.tsx:33]: by this, do you mean `location`? if so i believe yes.

**Review by johnschoeman — APPROVED** [review]
Removing dead code! nice!  LGTM, but i would definitely get eyes from someone familiar with the GPS flow.

File: app/views/onboarding/LocationsPermissions.tsx (hunk 1 above)
Comment (tparesi) [reply app/views/onboarding/LocationsPermissions.tsx:33]: Sorry, I meant to only remove the authSubscription here. Fixing!

Comment (tparesi) [reply app/views/onboarding/LocationsPermissions.tsx:33]: No, I meant the authSubscription inside the LocationsPermissions. Below, I kept the location stuff still.

**Review by JacobJaffe — APPROVED** [review]
Thanks for cleaning this up!

**Pull request merged**